src/app/containers/Preferences/General/index.jsx

```diff
@@ -23,6 +23,7 @@
 
 import React from 'react';
 import JoggingPresets from './JoggingPresets';
+import Aaxis from './Aaxis';
```

Comment:
Again would prefer a better naming convention, components included, this can be named "RotaryAxis". You would have to update it here and in the actual component itself